test/integration/eds_integration_test.cc

```diff
@@ -61,8 +61,9 @@ class EdsIntegrationTest : public testing::TestWithParam<Network::Address::IpVer
   void setEndpoints(uint32_t total_endpoints, uint32_t healthy_endpoints,
                     uint32_t degraded_endpoints, bool remaining_unhealthy = true,
                     absl::optional<uint32_t> overprovisioning_factor = absl::nullopt,
-                    bool await_update = true) {
+                    bool await_update = true, uint32_t disable_active_hc_endpoints = 0) {
```

Comment:
I think at this point, the number of integer positional parameters has gotten out-of-hand. I would make this an options struct so we can set defaults and clearly see what is being configured.

Reply:
Good point & changed to an option struct. Seems designated initializer is a C++20 feature, but we are not allowing to use that.